x-pack/plugins/infra/server/routes/log_alerts/chart_preview_data.ts

```diff
@@ -41,7 +41,8 @@ export const initGetLogAlertsChartPreviewDataRoute = ({
           resolvedLogView,
           framework.callWithRequest,
           alertParams,
-          buckets
+          buckets,
+          executionTimestamp
```

Comment:
What does `executionTimestamp` do?

Reply:
`executionTimestamp` is what we are going to use to tell the chart where to stop aka. `lte`. It's an optional field. If we pass nothing, its value `new Date.now()`